src/index.js

```diff
@@ -3,6 +3,18 @@ import ReactDOM from 'react-dom';
 import App from './App';
 import './index.css';
 
+import firebase from 'firebase';
+
+var config = {
+    apiKey: "AIzaSyCiksOUFfr8OR8ghevuDDzN0MtESAygBlA",
+    authDomain: "brews-clues.firebaseapp.com",
+    databaseURL: "https://brews-clues.firebaseio.com",
+    projectId: "brews-clues",
+    storageBucket: "brews-clues.appspot.com",
+    messagingSenderId: "33324339221"
+  };
+firebase.initializeApp(config);
```

Comment:
i literally lifted this all from the webpage, there is almost no original code here except for the `import` line.